### PR TITLE
BindingDesc BDlaethg28

### PR DESCRIPTION
**Pull request by ElianaDalSasso**
Hi everyone!
Sorry, I closed the previous pull request because I compared it to master -.-'

So, here is the small binding description of the Aeth g 28.
I added also a suggestion of modification and a comment to the ruling pattern and condition sections that were encoded by Sanne.

Thank you!

Eliana

### CODE REVIEW

**Review by gnizla — APPROVED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethg28.xml [hunk 1]
```diff
@@ -135,9 +138,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </dimensions>
                                     <note>The characters per line are 10–12.</note>
                                     <ab type="pricking">Ruling and pricking are barely visible.</ab>
-                                    <ab type="ruling" subtype="pattern">
+                                    <!--<ab type="ruling" subtype="pattern">
                                         <locus from="2r" to="15v"/>Ruling pattern:
-                                        1A-1A/0-0/0-0/C.</ab>
+                                        1A-1A/0-0/0-0/C.</ab> I don't see the the horizontal ruled lines, do you?
+                                      Shall we include the ruling pattern also for this record?-->
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethg28.xml:144]: maybe at the bottom of f. 10? But no, not really. Let's Massimo have a look as well.

**Review by thea-m — APPROVED** [review]
Thank you, that's great! I look forward to your next descriptions and improvements on existing parts of the description.

File: OxfordBodleian/Juel-Jensen/BDLaethg28.xml (hunk 1 above)
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethg28.xml:144]: I can see horizontal lines at the bottom of f. 3v (pic 12) and 6r (pic 14), but most folia seem to have no ruled lines.

**Review by MassimoVilla — APPROVED** [review]
Thank you Eliana. I am not merging in case you want to revise the ruling part.

**Pull request merged**